### DISCUSSION TITLE
Wrong metric priority on client wlan interface

### DISCUSSION THREAD

**Discussion by frranck** [RaspAP/raspap-webgui#1171] [Q&A]
Hello,
I did a default install and added a wifi client using an external wifi adapter.
by default the AP won't work because it gives a wrong priority to wlan1

```
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
default         10.3.141.1      0.0.0.0         UG    303    0        0 wlan0
default         fritz.box       0.0.0.0         UG    304    0        0 wlan1
10.3.141.0      0.0.0.0         255.255.255.0   U     303    0        0 wlan0
192.168.178.0   0.0.0.0         255.255.255.0   U     304    0        0 wlan1
```

If I run 

`sudo ifmetric wlan1 201
`

My AP will work as expected. 
Shouldn't wifi clients be given a lower priority? I am missing something in the configuration?

**Comment by billz** [comment]
Strictly speaking, the value is not "wrong" ... only that it has a default metric assigned by the DHCP daemon.

RaspAP doesn't presume to know your intensions with the new interface, so it doesn't apply arbitrary logic to set or otherwise modify metric values on the user's behalf. Metrics may be defined for all available interfaces in the **DHCP server** UI. The routing table is displayed on the **Networking > Summary** tab. 

In this two adapter "Wi-Fi repeater" setup, wlan0 (the AP interface) is configured like so:

```
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
0.0.0.0         192.168.1.254   0.0.0.0         UG    304    0        0 wlan1
10.3.141.0      0.0.0.0         255.255.255.0   U     999    0        0 wlan0
192.168.1.0     0.0.0.0         255.255.255.0   U     304    0        0 wlan1
```

Related discussion https://github.com/RaspAP/raspap-webgui/discussions/1022 and issue https://github.com/RaspAP/raspap-webgui/issues/335#issuecomment-541436840
